sdcm/cluster.py

```diff
@@ -1850,30 +1861,40 @@ def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):
             node.remoter.run('sudo yum install -y {}-tools'.format(node.scylla_pkg()))
         else:
             node.remoter.run('sudo apt-get update')
-            node.remoter.run('sudo apt-get install -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --force-yes --allow-unauthenticated {}-tools'.format(node.scylla_pkg()))
+            node.remoter.run('sudo apt-get install -y -o Dpkg::Options::="--force-confdef"'
+                             ' -o Dpkg::Options::="--force-confold" --force-yes'
+                             ' --allow-unauthenticated {}-tools'.format(node.scylla_pkg()))
 
-        node.wait_cs_installed(verbose=verbose)
         if node.is_rhel_like():
             node.remoter.run('sudo yum install -y screen')
         else:
             node.remoter.run('sudo apt-get install -y screen')
         if db_node_address is not None:
-            node.remoter.run("echo 'export DB_ADDRESS=%s' >> $HOME/.bashrc" %
-                             db_node_address)
-
-        cs_exporter_setup = CassandraStressExporterSetup()
-        cs_exporter_setup.install(node)
+            node.remoter.run("echo 'export DB_ADDRESS=%s' >> $HOME/.bashrc" % db_node_address)
 
-        if self.params.get('bench_run', default=False):
-            # go1.7 still not in repo
+        bench_run = self.params.get('bench_run', default=False)
+        gemini_run = self.params.get('gemini_run', default=False)
+        if bench_run or gemini_run:
```

Comment:
Can we replace these params with checking if gemini_cmd and bench_cmd?
Then our YAMLs don't need 2 params for the same behaviour.

Reply:
The command can be defined inside a test, so it's not a good way, I think.
In general, we can define something like cassandra-stress: False and install the both: scylla-bench and gemini